core/src/main/java/io/aiven/klaw/helpers/db/rdbms/HandleDbRequestsJdbc.java

```diff
@@ -911,6 +911,13 @@ public Map<String, Set<String>> getTopicAndVersionsForEnvAndTenantId(String envI
     return jdbcSelectHelper.getTopicAndVersionsForEnvAndTenantId(envId, tenantId);
   }
 
+  @Override
+  public MessageSchema getTeamIdFromSchemaNameAndEnvAndTenantId(
```

Comment:
```suggestion
  public MessageSchema getTeamIdFromSchemaTopicNameAndEnvAndTenantId(
```
so we make it everywhere unique